### ISSUE TITLE
e6 Suggestion / bug - IThermal Induction Smelter Automation of Cured Rubber (and other Induction Smelter recepies that require E.G. sand as a catalyst)

### ISSUE THREAD

**Issue by mjevans** [NillerMedDild/Enigmatica6#3908]
### Describe your suggestion

I'm trying to automate Cured Rubber production with Refined Storage, currently using the Induction Smelter.  In 1.16.5 the Induction Smelter appears to have only three modes that interact for item input: directionless (but accessible), input only, input and output (which might differ from directionless for auto pull/push?)

When I create an RS Processing recipe it puts sulfur, 2 dry rubber, and sand in order.  When it attempts to craft these three items enter the top three crafting slots and the recipe doesn't process (because it wants sand in the bottom slot).

I can't even just setup a dedicated Induction Smelter with a sand input, because this version of Thermal doesn't have isolated inputs like Mekanism machines still have.

### Why would you like this added/changed?

Suggestion: Add a recipe which works with automation.  Maybe there's a stick or some other item added so it fills all 4 blue input slots.  Maybe just accept the three items in the top slots (in addition to how Thermal wants it).

**Comment by MuteTiefling** [comment]
Yeah, it's trying to be smart here. Things that are catalysts should go in the bottom, while things that are recipes go to the top.

Sand can be both, unfortunately. In the next update you'll have slot locks, which will fix the issue for you. Until then, it seems your only recourse is to not use sand as a catalyst.

**Comment by NillerMedDild** [comment]
Still doesn't seem to be a thing in latest Thermal :(

**Comment by Darkere** [comment]
I've been using the slot seal. Definitely there.

**Comment by NillerMedDild** [comment]
I somehow don't see it 🤔

**Comment by Darkere** [comment]
![image](https://user-images.githubusercontent.com/4283717/155801603-17348db3-4c19-447a-a032-1d76b332bc16.png)

**Comment by NillerMedDild** [comment]
Oooh, I expected a per machine setting.
